surefire-its/src/test/java/org/apache/maven/surefire/its/fixture/MavenLauncher.java

```diff
@@ -186,9 +186,9 @@ public MavenLauncher addEnvVar( String key, String value )
         return this;
     }
 
-    public MavenLauncher assertNotPresent( String subFile )
+    public MavenLauncher verifyFileNotPresent( String subFile ) throws VerificationException
     {
-        getVerifier().assertFileNotPresent( getValidator().getSubFile( subFile ).getAbsolutePath() );
```

Comment:
Was this method renamed in new verifier?

Reply:
old method was deprecated
https://github.com/apache/maven-verifier/pull/9